draft-ietf-ppm-dap.md

```diff
@@ -3436,8 +3434,8 @@ the task's `time_precision` is 1000 seconds and the report's timestamp is
 
```

Comment:
The example timestamp on L3433 should be updated to match the interval start, I believe.

Reply:
Should it? I think the `report's timestamp` is different than the `Time` representation -- but maybe I could rephrase that to add `raw` or similar?

```diff
For example, if
- the task's `time_precision` is 1000 seconds and the report's timestamp is
+ the task's `time_precision` is 1000 seconds and the report's raw timestamp is
1729629081, the relevant batch bucket identifier is
```

Reply:
It's verbose, but how about "...is 1000 seconds and the report was generated at 1729629081 seconds after the start of the UNIX epoch, the relevant..."